views/homework/assignment.hbs

```diff
@@ -28,10 +28,10 @@
 {{> "lib/components/files-storage/imports"}}
 {{#extend "homework/homework"}}
     {{#content "page"}}
-        <section class="section-homework">
+        <section class="section-homework" data-testid="task-section-task">
             <div class="dates text-muted">
 				{{#if (userHasRole "student")}}
-				<i class="fa fa-{{#if submissions}}check done{{else if warning}}exclamation-triangle danger{{else}}{{#unless submittable}}ban closed{{/unless}}{{/if}}"></i>
+				<i class="fa fa-{{#if submissions}}check done{{else if warning}}exclamation-triangle danger{{else}}{{#unless submittable}}ban closed{{/unless}}{{/if}}" {{#if submissions}}data-testid="tasks-submission-hint-received"{{/if}}></i>
```

Comment:
Hier ist nun sehr viel Logik in disem i Tag. Ist es vielleicht besser die Fallunterscheidung nach außen zu nehmen und dann für Warning und Submisson einzelne i Tags zu rendern?